### PR TITLE
🧹 Add aws waf resource docs

### PR DESCRIPTION
**Pull request by mariuskimmina**

### CODE REVIEW

File: providers/aws/resources/aws.lr [hunk 1]
```diff
@@ -194,227 +194,366 @@ private aws.waf.rule.action @defaults("action") {
...
   json dict
+  // A statement for detecting sqli attacks
   sqliMatchStatement aws.waf.rule.statement.sqlimatchstatement
+  // A statement for detecting xss attacks
   xssMatchStatement aws.waf.rule.statement.xssmatchstatement
+  // A statement for matching certain bytes
   byteMatchStatement aws.waf.rule.statement.bytematchstatement
+  // A statement for matching a regex pattern
   regexMatchStatement aws.waf.rule.statement.regexmatchstatement
+  // A statement for matching requests from certain countrys
```
Comment (tas50) [inline providers/aws/resources/aws.lr]:
```suggestion
  // A statement for matching requests from certain countries
```

File: providers/aws/resources/aws.lr [hunk 2]
```diff
@@ -194,227 +194,366 @@ private aws.waf.rule.action @defaults("action") {
...
 
+// A statement that matches sqli attacks
 private aws.waf.rule.statement.sqlimatchstatement {
+  // the name of the rule this statement belongs to
   ruleName string
+  // the ID of the statement
   statementID string
+  // Describes the field that is being matched
   fieldToMatch aws.waf.rule.fieldtomatch
+  // Defines how aggresive the statement matches
```
Comment (tas50) [inline providers/aws/resources/aws.lr]:
```suggestion
  // Defines how aggressive the statement matches
```

File: providers/aws/resources/aws.lr [hunk 3]
```diff
@@ -194,227 +194,366 @@ private aws.waf.rule.action @defaults("action") {
...
   excludeRules []string
 }
 
 // Rule statement that checks the size of the specified field
 private aws.waf.rule.statement.sizeconstraintstatement {
+  // the name of the rule this statement belongs to
   ruleName string
+  // the ID of the statement
   statementID string
+  // The size that tirggers this statement
```
Comment (tas50) [inline providers/aws/resources/aws.lr]:
```suggestion
  // The size that triggers this statement
```

**Comment by tas50** [comment]
Hold off on merging this until @misterpantz get's all the fixes in

**Review by misterpantz — APPROVED** [review]
Thanks, @mariuskimmina! Great teamwork!

**Pull request merged**